src/runtimes/node/in_source_config/index.ts

```diff
@@ -84,10 +73,32 @@ const normalizeMethods = (input: unknown, name: string): string[] | undefined =>
   })
 }
 
-export const findISCDeclarations = (
+/**
+ * Loads a file at a given path, parses it into an AST, and returns a series of
+ * data points, such as in-source configuration properties and other metadata.
+ */
+export const parseFile = async (
```

Comment:
This was renamed from `findISCDeclarationsInPath`. No functionality has changed.